src/plugins/telemetry_collection_manager/server/types.ts

```diff
@@ -83,6 +83,10 @@ export interface BasicStatsPayload {
   cluster_stats: object;
   collection?: string;
   stack_stats: object;
+  elasticsearch?: {
+    cluster?: { id: string };
+    stats?: object;
+  };
```

Comment:
Yeah this was hard decision and the point is for safety. For example, `local_xpack` and `monitoring` plugins silently crash ([source](https://github.com/elastic/kibana/blob/main/src/plugins/telemetry_collection_manager/server/plugin.ts#L306-L308)). I will work on the dashboards after this fix and may re-address this if it needs to standardize the shape.

Reply:
FWIW, I don't think `telemetry_collection_manager` is validating anything inside `stack_stats` when in production (we do validate `stack_stats` against the schema in CI to make sure that we document the data structure and any changes in it). So I doubt we will silently crash because of a few extra fields (or clashing mismatching structures). Did I overlook anything?

If you are referring to "safety" as avoiding breaking changes in the shape of the data we report, I'm happy to resolve this conversation as it seems a duplicate to https://github.com/elastic/kibana/pull/182304/files#r1589552940